src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java

```diff
@@ -98,6 +99,12 @@ public NumericFieldSet getExtraNumericFields() {
     public void setExtraNumericFields(NumericFieldSet extraNumericFields) {
         this.extraNumericFields = extraNumericFields;
     }
+
+    public Date getLastAudited() {
```

Comment:
Ok, I'll move this line back to PlanGenerator. AFAIK there is already some NIH specific stuff where this method gets called. 

Will also add a TODO for clarity.